ddtrace/tracer.py

```diff
@@ -179,8 +179,7 @@ def _default_span_processors_factory(
     if single_span_sampling_rules:
         span_processors.append(SpanSamplingProcessor(single_span_sampling_rules))
 
-    if peer_service_processor.enabled:
```

Comment:
this change causes us to affect existing behavior.  Are there any current uses of peer.service this would affect (is peer.service used for anything?) If so, we'll need a release note and to revisit the decision

Reply:
it is not used for anything today product-wise so my advice is not to advertise (Also because we're not going to advertise the peer.service remapping option)